luarules/gadgets/unit_gunship_strafe_fix.lua

```diff
@@ -0,0 +1,100 @@
+local gadget = gadget ---@type Gadget
+
+function gadget:GetInfo()
+	return {
+		name = "Gunship Strafe Fix",
+		desc = "When gunships stop attacking their target unit with no followup command, forces them to stop strafing and enter idle behavior",
+		author = "JRTaylord: https://github.com/JRTaylord",
+		date = "May 7, 2025",
+		license = "GNU GPL, v2 or later",
+		layer = 0, -- Todo: ask which layer should this be on
```

Comment:
Should be on 0 unless there is a known reason not to.

Reply:
Perfect, thanks